documentation/collaboration.md

```
@@ -0,0 +1,36 @@
+# Collaboration 
+
+By default with Docs, collaboration is enabled. To allow the collaboration between user, a connection to a websocket server is made (the y-provider service), you only have to configure the Django backend URL in your y-provider service:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)